src/components/TextField.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class TextField extends React.Component {
+  render() {
+    const {
+      value,
+      onChange,
+      onFocus,
+      onBlur,
+      label,
+      placeholder,
+      type,
+      multiline,
+      classes = defaultClasses
```

Comment:
pass down rest of the props to the component..
```javacript
const {
      value,
      onChange,
      onFocus,
      onBlur,
     ...rest
 } = this.props

In your component
<input
              className={input}
              label={label}
              onChange={onChange}
              onBlur={onBlur}
              onFocus={onFocus}
              type={type}
              value={value}
              placeholder={placeholder}
             {...rest}
            />
```

Reply:
noted

Reply:
Yes sorry my bad, However we might have to do ref forwarding right?